samples/src/main/java/org/ishugaliy/allgood/consistent/hash/samples/HasherLoadDistributionSandbox.java

```diff
@@ -13,12 +13,39 @@
 import java.util.stream.Collectors;
 import java.util.stream.IntStream;
 
+/**
+ * Sandbox allows to check consistent hash load distribution between nodes
+ * with different hash functions. Showing how distribution depends on hash function.
```

Comment:
Sandbox allows checking consistent hash load distribution between nodes with different hash functions. Showing how distribution depends on the hash function.